plugin/pkg/admission/resourcequota/config_test.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcequota
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestLoadConfiguration(t *testing.T) {
```

Comment:
Wondering whether we can move the boilerplate into a shared place and only pass-in a valid config and the plugin? (the plugin initializer receives the  `io.Reader`).

Reply:
I am also confused.

Reply:
Would like to see a `config_test.go` for every plugin with config. But then we should reduce the boilerplate.

Reply:
Which boilerplate can be reduced?

Reply:
Imagine we want this kind of test 7 times?

Reply:
ic, will think about.